src/module.c

```diff
@@ -8231,9 +8260,12 @@ void *RM_DefragAlloc(RedisModuleDefragCtx *ctx, void *ptr) {
 }
 
 /* Defrag a RedisModuleString previously allocated by RM_Alloc, RM_Calloc, etc.
- * Returns a newly allocated pointer, or NULL if defrag is not required.
+ * See RM_DefragAlloc() for more information on how the defragmentation process
+ * works.
+ *
+ * NOTE: Currently retained strings (which have multiple references) do not get
+ * defragmented.
```

Comment:
this may be misleading since in many cases strings that arrive from the argv and are retained actually have only one reference count.
not sure how to improve..
also the word "currently" may not be needed.

Reply:
Tried to improve that.